src/smartpeak/source/core/RawDataProcessor.cpp

```diff
@@ -1641,7 +1703,60 @@ namespace SmartPeak
     OpenMS::SavitzkyGolayFilter sgfilter;
     OpenMS::PeakPickerHiRes picker;
     OpenMS::PeakIntegrator pi;
-    return ParameterSet({ sgfilter, picker, pi });
+    ParameterSet parameters({ sgfilter, picker, pi });
+
+    std::map<std::string, std::vector<std::map<std::string, std::string>>> param_struct({
+    {"PickMS1Features", {
+    {
+      {"name", "frame_length"},
+      {"type", "int"},
+      {"value", "11"},
+      {"description", "SavitzkyGolayFilter parameter. The number of subsequent data points used for smoothing"},
+    },
+    {
+      {"name", "polynomial_order"},
+      {"type", "int"},
+      {"value", "4"},
+      {"description", "SavitzkyGolayFilter parameter. Order or the polynomial that is fitted"},
+    },
+    {
+      {"name", "sne:window"},
+      {"type", "float"},
+      {"value", "10"},
+      {"description", "SignalToNoiseEstimatorMedianRapid parameter. Signal-to-noise estimation window (in mz)"},
+    },
+    {
+      {"name", "write_convex_hull"},
+      {"type", "bool"},
+      {"value", "false"},
+      {"description", "Whether to write out all points of all features into the featureXML"},
+    },
+    {
+      {"name", "compute_peak_shape_metrics"},
+      {"type", "bool"},
+      {"value", "false"},
+      {"description", "Calulates various peak shape metrics (e.g., tailing) that can be used for downstream QC/QA."},
+    },
+    {
+      {"name", "min_intensity"},
+      {"type", "float"},
+      {"value", "86000"},
+      {"description", "All features below the minimum intensity will be discarded"},
+    },
+    {
+      {"name", "baseline_type"},
+      {"type", "string"},
+      {"value", "vertical_division"},
```

Comment:
The description for these entries are the same as for the parameters of the PeakPickerMRM.

Reply:
actually i feel these 2 baseline_type and integration_type should not be listed here as they are not used later in the code, and they are defined in openms PeakIntegrator, which is included in the PickMS1Features::getParameterSchema

Reply:
If that is the case then please just remove them.